application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadAutoArchiver.java

```diff
@@ -122,16 +127,30 @@ private static boolean shouldBeArchived(MessageChannel channel, Instant archiveA
     }
 
     private void handleArchiveFlow(ThreadChannel threadChannel, MessageEmbed embed) {
+
         Consumer<Throwable> handleFailure = error -> {
             if (error instanceof ErrorResponseException) {
                 logger.warn("Unknown error occurred during help thread auto archive routine",
                         error);
             }
```

Comment:
if something breaks, should we archive anyway?

Reply:
yes ig